playwright-tests/api/settingsApiV1.ts

```diff
@@ -0,0 +1,29 @@
+import apiHelper, {Settings} from "@api/helpers/apiHelper";
+import {APIResponse} from "playwright-core";
+
+const PATH_GET = 'v1/Settings/Get';
+const PATH_CHANGE = 'v1/Settings/Get';
+
+export enum SettingProperty {
+  bm = "backup_management_enabled",
+}
+
+export const settingsAPIv1 = {
```

Comment:
```suggestion
export const settingsAPI = {
```

Reply:
Why do you want to exclude the API version from the file name? This important detail shows given API origin - this is a client facing api and its' version is 1

Reply:
fixed